schema.ts

```diff
@@ -1201,7 +1211,24 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * @hidden
    */
   modifiedByProperty?: PropertyIdentifier<K>;
-
+  /**
+   * For cases where the object being synced represents a user, the name of the property within
+   * {@link ObjectSchemaDefinition.properties} that identifies the email address of the user.
+   *
+   * Must be a {@link ValueType.String} property with the {@link ValueHintType.Email} hint or
+   * a {@link ValueType.Object} with the {@link ValueHintType.Person} hint
+   * @hidden
+   */
+  userEmailProperty?: PropertyIdentifier<K>;
+  /**
+   * For cases where the object being synced represents a user, the name of the property within
+   * {@link ObjectSchemaDefinition.properties} that identifies the id of the user in the service
+   * being synced from.
+   *
+   * Must be a {@link ValueType.String} or {@link ValueType.Number} property
```

Comment:
Would this allow for properties with the hint `Email` to be used as well? I imagine in some systems the only user ID is the user email.

Reply:
Yeah this would allow that